src/flamenco/runtime/fd_runtime.c

```diff
@@ -3035,46 +3035,40 @@ fd_runtime_calculate_fee(fd_exec_txn_ctx_t *txn_ctx,
 #define FD_RENT_EXEMPT (-1L)
 
 static long
-fd_rent_due(fd_account_meta_t *acc,
-            ulong epoch,
-            fd_rent_t const *rent,
-            fd_epoch_schedule_t const *schedule,
-            double slots_per_year)
-{
+fd_rent_due(fd_account_meta_t *         acc,
+            ulong                       epoch,
+            fd_rent_t const *           rent,
+            fd_epoch_schedule_t const * schedule,
+            double                      slots_per_year) {
 
   fd_solana_account_meta_t *info = &acc->info;
 
-  /* Nothing due if account is rent-exempt */
-
-  ulong min_balance = fd_rent_exempt_minimum_balance2(rent, acc->dlen);
-  if (info->lamports >= min_balance)
-  {
+  /* Nothing due if account is rent-exempt
+     https://github.com/anza-xyz/agave/blob/v2.0.9/sdk/src/rent_collector.rs#L90 */
+  ulong min_balance = fd_rent_exempt_minimum_balance2( rent, acc->dlen );
+  if( info->lamports >= min_balance ) {
     return FD_RENT_EXEMPT;
   }
 
-  /* Count the number of slots that have passed since last collection */
-
+  /* Count the number of slots that have passed since last collection 
+     https://github.com/anza-xyz/agave/blob/v2.0.9/sdk/src/rent_collector.rs#L93-L98 */
```

Comment:
lets use `//` for permalinks instead of `/* */`

Reply:
disagree